addons/mail/static/src/components/mobile_messaging_navbar/mobile_messaging_navbar.js

```diff
@@ -15,41 +22,20 @@ export class MobileMessagingNavbar extends Component {
      * @param {MouseEvent} ev
      */
     _onClick(ev) {
-        if (this.props.onSelectMobileMessagingNavbarTab) {
-            this.props.onSelectMobileMessagingNavbarTab({
-                tabId: ev.currentTarget.dataset.tabId,
-            });
-        }
+        this.mobileMessagingNavbarView.onClick(ev.currentTarget.dataset.tabId);
```

Comment:
move handler to model only, replace `dataset` by using `bind` in template like `onClickSelectablePartner` to pass a param